jwst/lastframe/lastframe_sub.py

```diff
@@ -40,7 +38,7 @@ def do_correction(input_model):
     # the final group to 'DO_NOT_USE'
     if sci_ngroups > 2:
         output.groupdq[:, -1, :, :] = \
-            np.bitwise_or(output.groupdq[:,-1,:,:], dqflags.group['DO_NOT_USE'])
+            np.bitwise_or(output.groupdq[:, -1, :, :], dqflags.group['DO_NOT_USE'])
```

Comment:
I think this line was fine before.  Slicing is a special case were you don't need space for readability.  If you're going to add space, it's good to have it symmetric.  See https://legacy.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements